docs/docs/themes/using-a-gatsby-theme.md

```diff
@@ -2,71 +2,55 @@
 title: Using a Gatsby Theme
 ---
 
-The quickest way to get started using a Gatsby theme is to use a starter that's configured to use the theme.
+While you can [get started quickly with a Gatsby theme starter](/docs/themes/getting-started/), you can also install a Gatsby theme directly to an existing Gatsby site. Gatsby themes are plugins, so we [use them just like any other Gatsby plugin](/docs/using-a-plugin-in-your-site/).
 
-For example, `gatsby-starter-blog-theme` is a theme starter for the `gatsby-theme-blog` package.
+## Installing a Theme
 
-A **regular Gatsby starter** creates a new Gatsby site that is preconfigured for a particular use case. The resulting site effectively forks off the starter — after it’s created, the site maintains no connection to the starter.
+Like any Gatsby plugin, Gatsby themes are Node.js packages, so you can install them like other packages in node using npm.
```

Comment:
```suggestion
Like any Gatsby plugin, Gatsby themes are Node.js packages, so you can install them like other published packages in Node using npm or [yarn, including local workspaces](#using-yarn-workspaces).
```

We don't typically mention Yarn but it is relevant to building themes, so it could be useful here.